include/ipr/traversal

```diff
@@ -9,6 +9,7 @@
 #define IPR_TRAVERSAL_INCLUDED
 
 #include <ipr/interface>
+#include <ipr/impl>
```

Comment:
Debatable whether <ipr/traversal> should depend on the default implementation. The IPR is designed in such a way that a client could write their own impl from scratch base on <ipr/interface>. In that scenario traversal utils are now not usable. Consider including this in impl itself if needed.

Reply:
Good point, maybe putting this in impl makes sense.